src/splitInteger.test.js

```diff
@@ -19,3 +19,10 @@ test('should sort parts ascending if they are not equal', () => {
 test('should add zeros if value < numberOfParts', () => {
 
 });
+
+test('should split into equal parts if divisible', () => {
```

Comment:
Add a brief description for the test suite 'should split into equal parts if divisible'.